packages/react-menu/src/components/MenuItemRadio/MenuItemRadio.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { useMenuItemRadio } from './useMenuItemRadio';
+import { MenuItemRadioProps } from './MenuItemRadio.types';
+import { renderMenuItemRadio } from './renderMenuItemRadio';
+import { useMenuItemStyles } from '../MenuItem/useMenuItemStyles';
+import { useCheckmarkStyles } from '../../selectable/index';
+
+/**
+ * Define a styled MenuItemRadio, using the `useMenuItemRadio` hook.
+ * {@docCategory MenuItemRadio}
+ */
+export const MenuItemRadio = React.forwardRef<HTMLElement, MenuItemRadioProps>((props, ref) => {
+  const state = useMenuItemRadio(props, ref);
+  useMenuItemStyles(state);
+  useCheckmarkStyles(state);
```

Comment:
_It will be interesting to see how this part is reused 💭_

Reply:
hmm that's a good point, let me reuse this in specific styling hooks

Reply:
I've created `useMenuItemCheckboxStyles` and `useMenuItemRadioStyles` that will reuse the checkmark styles, that should fix the reusability